### PR TITLE
hub: Add package, version 2.2.1

### PR DESCRIPTION
**Pull request by oysteinkrog**

### CODE REVIEW

File: mingw-w64-hub/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Maintainer: Øystein Krog <oystein.krog@gmail.com>
+
+_realname=hub
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=2.2.1
+pkgrel=1
+pkgdesc="hub introduces git to GitHub"
```
Comment (mingwandroid) [inline mingw-w64-hub/PKGBUILD]: Please add (mingw-w64) here

Comment (oysteinkrog) [inline mingw-w64-hub/PKGBUILD]: Sorry, I'm not sure I understand.
Do you want the description changed?

Comment (Alexpux) [inline mingw-w64-hub/PKGBUILD]: Yeah, add "(mingw-w64)" at the end of description

**Comment by Alexpux** [comment]
I will merge it but GO package is on WIP stage.

**Comment by oysteinkrog** [comment]
Ok, I have been using hub built using this since yesterday and so far it has been working well.

**Pull request merged**